### PR TITLE
Improve Readability

### PR DESCRIPTION
**Pull request by nilederg**
Just some simple changes to make the code easier to work with

### CODE REVIEW

File: src/main/PlagueMain.java [hunk 1]
```diff
@@ -1078,7 +1081,7 @@ void loadMap(String args[]) {
         Log.info("Loading map " + map.name());
 
         world.loadMap(map);
-        loadedMap = state.map;
+        mindustry.maps.Map loadedMap = state.map;
```
Comment (bend-n) [inline src/main/PlagueMain.java]: ew use the small signature

File: src/main/PlagueMain.java [hunk 2]
```diff
@@ -175,15 +180,13 @@ public void init() {
...
-                    return false;
-                }
+                return action.block == null ||
+                        ((!PlagueData.plagueBanned.contains(action.block) || !hasWon)
+                                &&
+                                (!PlagueData.plagueBannedPreWin.contains(action.block) || hasWon))
+                        || action.player.team() != Team.malis;
 
-            }
+            } // TODO readability
```
Comment (bend-n) [inline src/main/PlagueMain.java]: cant you fix this now?

**Review by bend-n — COMMENTED** [review]
have you tested it

File: src/main/PlagueMain.java (hunk 1 above)
Comment (nilederg) [reply src/main/PlagueMain.java]: I don't understand what you're asking me to do here. Can you elaborate?

Comment (bend-n) [reply src/main/PlagueMain.java]: `Map loadedMap = state.map`
Or, `String mapID = state.map.file.name();` because thats the only place its ever used

Comment (nilederg) [reply src/main/PlagueMain.java]: Map is already a reserved token in Java. It has to stay this way.

Comment (bend-n) [reply src/main/PlagueMain.java]: Then do the other option, or remove the type, your not increasing readability.

File: src/main/PlagueMain.java (hunk 2 above)
Comment (nilederg) [reply src/main/PlagueMain.java]: Complete.

File: src/main/PlagueMain.java [hunk 3]
```diff
@@ -638,46 +641,46 @@ public void registerClientCommands(CommandHandler handler) {
...
                         return;
                     }
 
                     if (args.length != 0) {
-                        for (CustomPlayer other : pTeam.players) {
-                            if (("" + other.player.id()).equals(args[0]) ||
-                                    other.player.name.equalsIgnoreCase(args[0]) ||
-                                    other.rawName.equalsIgnoreCase(args[0])) {
-                                if (other.player == player)
+                        for (CustomPlayer teammate : playerTeam.players) {
```
Comment (bend-n) [inline src/main/PlagueMain.java]: case

Comment (nilederg) [reply src/main/PlagueMain.java]: I don't see how this is an issue, seems readable enough to me

Comment (bend-n) [reply src/main/PlagueMain.java]: `teamMate`
even if your not in java `teammate` is invalid casing

Comment (nilederg) [reply src/main/PlagueMain.java]: teammate is one word

File: src/main/PlagueMain.java (hunk 1 above)
Comment (nilederg) [reply src/main/PlagueMain.java]: done

**Review by bend-n — CHANGES_REQUESTED** [review]
teamMate fight me

**Review by bend-n — APPROVED** [review]
i am happy

**Pull request merged**